src/Microsoft.ML.Data/Evaluators/BinaryClassifierEvaluator.cs

```diff
@@ -172,13 +172,13 @@ private protected override Aggregator GetAggregatorCore(RoleMappedSchema schema,
         private ReadOnlyMemory<char>[] GetClassNames(RoleMappedSchema schema)
         {
             // Get the label names if they exist, or use the default names.
-            ColumnType type;
             var labelNames = default(VBuffer<ReadOnlyMemory<char>>);
-            if (schema.Label.Type.IsKey &&
-                (type = schema.Schema[schema.Label.Index].Metadata.Schema.GetColumnOrNull(MetadataUtils.Kinds.KeyValues)?.Type) != null &&
-                type.ItemType.IsKnownSizeVector && type.ItemType.IsText)
```

Comment:
While most of the changes in the PR are intended to be over identical functionality, this was an exception since it was clearly a bug... `type.ItemType.IsKnownSizeVector` must always be false, since an item type could never be a vector type of any sort, known size or not!